CONTRIBUTING.md

```diff
@@ -74,6 +74,24 @@ You can use IntelliJ IDEA (preferred) or VS Code (alternate) in the browser.
 If you prefer using IntelliJ IDEA, you can setup Gitpod integration with JetBrains Gateway using the instructions on [gitpod.io](https://www.gitpod.io/docs/ides-and-editors/intellij),
 which will open the workspace in IntelliJ IDEA using JetBrains Gateway.
 
+### Linting
+
+For linting we use a number of tools:
+
+- eslint
+- stylelint
+- prettier
+- spotless
```

Comment:
```suggestion
- [eslint](https://eslint.org/)
- [stylelint](https://stylelint.io/)
- [prettier](https://prettier.io/)
- [spotless](https://github.com/diffplug/spotless)
```
Any objections about linking the tools? I doubt, everyone knows all linters, especially the front end ones.

Reply:
sure I thought about it but wasn't sure, happy to link